### PR TITLE
fixes #695

### LINKED ISSUE #695: Add note about persistent storage to CRUD mission
Add note about persistent storage to:

docs/topics/crud-mission-design-tradeoffs.adoc

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

**Comment by rhoads-zach** [comment]
@Ladicek mind taking a look?

File: docs/topics/crud-mission-design-tradeoffs.adoc [hunk 1]
```diff
@@ -5,9 +5,5 @@
 a| * Each runtime decides how the database interactions are implemented. One can use JDBC while others can use JPA or access ORM APIs directly. Each runtime decides what would be the best way.
 
  * Each runtime decides how the schema is going to be created.
-a|
-
-// not sure this has to be here.
-// is the database application scalable in the Missions?
- * The example PostgreSQL database provided with the Relational Database Backend Missions is not backed up with persistent storage. Changes to the database are lost if you stop or redeploy the database application.
+a| * The example PostgreSQL database provided with the Relational Database Backend Missions is not backed up with persistent storage. Changes to the database are lost if you stop or redeploy the database application. The OpenShift documentation has more link:https://docs.openshift.com/online/architecture/additional_concepts/storage.html[background information on persistent storage] as well as link:https://docs.openshift.com/online/dev_guide/volumes.html[more details on configuring volumes so containers can use persistent storage].
```
Comment (Ladicek) [inline docs/topics/crud-mission-design-tradeoffs.adoc]: Both links are valid and useful, but they are more useful for OpenShift cluster administrator than for a DevOps guy who wants to deploy a database. This link: https://docs.openshift.com/online/dev_guide/persistent_volumes.html shows how to use persistent volume claims, which is what the DevOps guys in question needs to do.

**Comment by rhoads-zach** [comment]
thank you for the feedback @Ladicek, updated to include your link.

**Comment by Ladicek** [comment]
Thanks, LGTM now.

**Comment by inoxx03** [comment]
merging...

**Comment by tradej** [comment]
I have a few stylistic suggestions, mainly regarding consistency with other parts of the docs. PR imminent...

**Comment by tradej** [comment]
See https://github.com/rhoads-zach/documentation/pull/5 .

**Comment by rhoads-zach** [comment]
@tradej thank you for the feedback. I've merged your changes.

**Comment by inoxx03** [comment]
Latest changes look good. Merging...

**Comment by inoxx03** [comment]
merged: b66ef23fd44f6dbce9024b9a70b1f80f79732af0

**Pull request closed without merging**